app/src/main/java/eu/kanade/tachiyomi/data/backup/BackupManager.kt

```diff
@@ -526,6 +527,8 @@ class BackupManager(
                 thumbnailUrl = manga.thumbnailUrl,
                 favorite = manga.favorite.toLong(),
                 lastUpdate = manga.lastUpdate,
+                nextUpdate = 0L,
+                calculateInterval = 0L,
```

Comment:
```suggestion
                nextUpdate = null,
                calculateInterval = null,
```
So it doesn't overwrite db value

Reply:
Updated